launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml

```diff
@@ -367,14 +367,15 @@
     <include file="$(find-pkg-share object_merger)/launch/object_association_merger.launch.xml">
       <arg name="input/object0" value="$(var merger/input/objects)"/>
       <arg name="input/object1" value="clustering/camera_lidar_fusion/objects"/>
-      <arg name="output/object" value="$(var lidar_detection_model)_roi_cluster_fusion/objects"/>
+      <!-- <arg name="output/object" value="$(var lidar_detection_model)_roi_cluster_fusion/objects"/> -->
+      <arg name="output/object" value="(var merger/output/objects)"/>
```

Comment:
```suggestion
      <arg name="output/object" value="objects_before_filter"/>
```

Reply:
this will fixed the ERROR: 
`[ERROR] [object_association_merger_node-72]: process has died [pid 17557, exit code -6, cmd '/home/autoware/j6.x2.pilot/install/object_merger/lib/object_merger/object_association_merger_node `